tests/setup_module_utils.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+set -euo pipefail
+
+if [ -n "${DEBUG:-}" ] ; then
+    set -x
+fi
+
+if [ ! -d "${1:-}" ] ; then
+    echo Either ansible is not installed, or there is no ansible/module_utils
+    echo in $1 - Skipping
+    exit 0
+fi
+
+if [ ! -d "${2:-}" ] ; then
+    echo Role has no module_utils - Skipping
```

Comment:
Same problem as the check for `$1`.

Reply:
I wanted to allow for this script being called separately in the future, in which case I wanted to leave the argument checking in.  It will also catch the case where someone makes an inadvertent breaking change to `tox.ini` where the environment variables are set.  So, I would prefer to leave this as-is.